lldb/source/Plugins/Process/Dpu/Dpu.cpp

```diff
@@ -487,9 +499,49 @@ bool Dpu::WriteWRAM(uint32_t offset, const void *buf, size_t size) {
                                  final_size_in_dpuword);
 
   delete[] words;
+  // printf("%s:%d:%s return res = %d\n", __FILE__, __LINE__, __func__, ret);
   return ret == DPU_OK;
 }
 
+bool Dpu::WriteWRAM(uint32_t offset, const void *buf, size_t size) {
+  return this->WriteWRAMTreatingAlignement(offset, buf, size);
```

Comment:
I assume everything after that is leftover debug? Or is this early return the debug state?